run-tests.sh

```diff
@@ -42,34 +42,9 @@ if which yum; then
     sudo yum -y install redhat-lsb-core epel-release
 fi
 
-# Get a list of packages to install with bindep. If packages need to be
-# installed, bindep exits with an exit code of 1.
-BINDEP_PKGS=$(bindep -b -f bindep.txt test || true)
-echo "Packages to install: ${BINDEP_PKGS}"
-
-# Install a list of OS packages provided by bindep.
-if which apt-get; then
-    sudo apt-get update
-    DEBIAN_FRONTEND=noninteractive \
-      sudo apt-get -q --option "Dpkg::Options::=--force-confold" \
-      --assume-yes install $BINDEP_PKGS
-elif which yum; then
-    # Don't run yum with an empty list of packages.
-    # It will fail and cause the script to exit with an error.
-    if [[ ${#BINDEP_PKGS} > 0 ]]; then
-      sudo yum install -y $BINDEP_PKGS
-    fi
+if [ "${FUNCTIONAL_TEST}" = true ]; then
+  tox -e bindep
+  tox -e functional
```

Comment:
This is clear, I love it. I'd rather do that into a separate commit though, to be able to reapply everywhere else in OSA.